Client/core/CCommandFuncs.cpp

```diff
@@ -47,10 +41,6 @@ void CCommandFuncs::Exit(const char* szParameters)
 
 void CCommandFuncs::Ver(const char* szParameters)
 {
-    ShowCursor(TRUE);
-    HCURSOR hc = LoadCursor(NULL, IDC_ARROW);
-    SetCursor(hc);
-
```

Comment:
These changes come from initial commit, seems reasonable to remove 👍

Reply:
Merged in 0cdc4d2d53dd7f829767e1f790ea47b29fc1ff31.